api/common/v1alpha1/ref.go

```diff
@@ -56,3 +56,12 @@ func (in *TargetRef) Hash() string {
 	}
 	return fmt.Sprintf("%s/%s/%s/%s", in.Kind, in.Name, strings.Join(orderedTags, "/"), in.Mesh)
 }
+
+// BackendRef defines structure that allows describing weighted attaching point
+// to various objects
```

Comment:
This comment shows up in the CRDs.

```suggestion
// BackendRef defines where to forward traffic.
```